cohttp/test/dune

```diff
@@ -1,6 +1,7 @@
 (executable
  (name test_accept)
  (modules test_accept)
+ (forbidden_libraries base)
```

Comment:
Do we need to update dune lower bound to use this?

Reply:
good point, I've just bumped it in the opam file (the dune-project file was already using the right version)